ServiceAreaParkingStatus/Presentation/Main/ViewController.swift

```diff
@@ -194,13 +196,26 @@ extension ViewController: ParkingManagerDelegate {
         parkingManager.delegate = self
         parkingManager.fetchParking()
     }
+    
+    func changeDataFormat(serviceAreaNameArray: [String]) {
```

Comment:
`return`을 해주는 함수가 아니기 때문에 어떤 타입으로 변환해주는 건지 알기 어렵습니다. 주석을 추가하거나 메서드 이름만으로 알수 있게 수정하는게 좋습니다

그리고 해당 메서드는 데이터를 변환해 새로운 데이터를 만드는 동작이기 때문에 메서드 내부에서 직접 self의 프로퍼티를 수정하는 것 보다는, 완전한 데이터를 `return` 해 주는 방식이 재사용하거나 사이드 이슈를 방지하는데 좋습니다

```suggestion
    /// 지역 휴게소 이름을 [지역이름: 방면 이름] 형식으로 변환
    func changeDataFormat(serviceAreaNameArray: [String]) -> [String: String] {
```